src/strSlice.mli

```diff
@@ -30,6 +36,10 @@ val drop_last : t -> t
 val take : int -> t -> char list
 (** [take n s] is a list of the first [n] characters of [s] *)
 
+val take_n : int -> t -> t
+(** [take_n n s] returns the slice consisting of the first [n]
+    characters of [s]. *)
```

Comment:
Just to be consistent with the "equational idiom" used in the rest of the docs for this module:

```suggestion
(** [take_n n s] is the slice consisting of the first [n]
    characters of [s]. *)
```